Runtime/BlockadeLabsSkyboxMesh.cs

```diff
@@ -110,11 +111,6 @@ private void OnDisable()
         }
 
         public int? GetRemixId()
-        {
-            return GetMetadata()?.request?.id;
-        }
-
-        internal GetImagineResult GetMetadata()
         {
             if (!TryGetComponent<Renderer>(out var renderer) || renderer.sharedMaterial == null || renderer.sharedMaterial.mainTexture == null)
```

Comment:
can probably remove this

Reply:
I thought we needed to know if the manTexture was set to `default_skybox_texture`?